packages/devtools_app/lib/src/screens/memory/panes/profile/profile_view.dart

```diff
@@ -235,6 +439,27 @@ class AllocationProfileTableViewState
             allocationProfileController: widget.controller,
           ),
         ),
+        ValueListenableBuilder<bool>(
+          valueListenable: preferences.vmDeveloperModeEnabled,
+          builder: (context, vmDeveloperModeEnabled, _) {
+            if (!vmDeveloperModeEnabled) return const SizedBox.shrink();
+            return Column(
+              children: [
+                SizedBox(
+                  // _GCStatsTable is a table with two header rows (column groups
+                  // and columns) and one data row. We add a slight padding to
```

Comment:
can we use a Padding widget to add this padding so that it is more clear what the padding value is?

Reply:
Maybe padding is the wrong word here, but that doesn't accomplish what this extra 1 pixel does. The child of `SizedBox` needs an extra pixel of space to disable the scrolling behavior of the underlying table. Padding will just add the extra pixel around the child widget without disabling the scroll behavior.